src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.model;
+
+public class Car {
+    private int movingDistance;
+
+    public Car() {
+        this(0);
+    }
+
+    private Car(int movingDistance) {
+        this.movingDistance = movingDistance;
+    }
+
+    public void move(boolean isCarMovingForward) {
+        if(isCarMovingForward)
+            movingDistance++;
```

Comment:
```suggestion
        if (isCarMovingForward) {
            movingDistance++;
        }    
```
띄어쓰기도 컨벤션 중 하나입니다.
그리고 한줄인 상황에서는 if 블록을 생략하기도 하는데요
저는 개인적으로 일관성있게 블록을 선언하는 것을 선호합니다.